application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -65,14 +76,22 @@ public RemindCommand(Database database) {
                 "period to remind you in, the unit of time (e.g. 5 [weeks])", true);
         TIME_UNITS.forEach(unit -> timeUnit.addChoice(unit, unit));
 
-        getData().addOptions(timeUnit, timeAmount)
-            .addOption(OptionType.STRING, CONTENT_OPTION, "what to remind you about", true);
+        getData().addSubcommands(
+                new SubcommandData(CREATE_SUBCOMMAND, "creates a reminder").addOptions(timeAmount,
+                        timeUnit,
+                        new OptionData(OptionType.STRING, CONTENT_OPTION,
+                                "what to remind you about", true)),
+                new SubcommandData(LIST_SUBCOMMAND, "shows all your currently pending reminders"));
```

Comment:
Maybe put the subcommands and options into seperate variables first. This looks really messy.